src/js/chrome/chrome_shim.js

```
@@ -908,5 +908,29 @@ module.exports = {
       }
       return e;
     });
+  },
+
+  shimGetDisplayMedia: function(window, getSourceId) {
+    if ('getDisplayMedia' in window.navigator) {
+      return;
+    }
+    // getSourceId is a function that returns a promise resolving with
+    // the sourceId of the screen to be shared.
+    if (!getSourceId) {
+      return;
```

Maybe throw an error instead of failing silently here? Also, maybe
```js
if (typeof getSourceId != "function") {
  return;
```
to proof it a bit against pilot error, since this arg is sometimes a function and sometimes a string?

---

Went for console.error